logstash-input-webfetcher/src/main/java/eu/wajja/web/fetcher/model/LogstashJson.java

```diff
@@ -0,0 +1,71 @@
+package eu.wajja.web.fetcher.model;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+
+/**
+ * JSON object that will be sent to logstash
+ * 
+ * @author mahytom
+ *
+ */
+@JsonInclude(Include.NON_NULL)
+public class LogstashJson implements Serializable {
```

Comment:
have a look at https://github.com/logstash/logstash-logback-encoder#event-specific-custom-fields
There should be no need for a custom object